content/docs/intro/concepts/_index.md

```diff
@@ -9,49 +9,292 @@ menu:
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts/resources" >}}"><i class="fas fa-server pr-2"></i>Resources</a></h3>
+        <p>Learn more about how to use and manage resources in your program.</p>
     </div>
 </div>
 <div class="md:flex flex-row mt-6 mb-6">
     <div class="w-1/2 border-solid border-t-2 border-gray-200">
-        <h3 class="no-anchor pt-4"><a href="{{< relref "stack" >}}"><i class="fas fa-cloud pr-2"></i>Stacks</a></h3>
-        <p>Pulumi's fundamental unit of partitioning, configuring, and scaling environments.</p>
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts/inputs-outputs" >}}"><i class="fas fa-hdd pr-2"></i>Inputs and Outputs</a></h3>
+        <p>Learn how to use resource properties to handle dependencies between resources.</p>
     </div>
     <div class="w-1/2 border-solid ml-4 border-t-2 border-gray-200">
-        <h3 class="no-anchor pt-4"><a href="{{< relref "config" >}}"><i class="fas fa-keyboard pr-2"></i>Configuration and Secrets</a></h3>
-        <p>How to configure stacks, including storing secret encrypted settings.</p>
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts/stack" >}}"><i class="fas fa-cloud pr-2"></i>Stacks</a></h3>
+        <p>Learn how to create and deploy stacks.</p>
     </div>
 </div>
 <div class="md:flex flex-row mt-6 mb-6">
     <div class="w-1/2 border-solid border-t-2 border-gray-200">
-        <h3 class="no-anchor pt-4"><a href="{{< relref "organizing-stacks-projects" >}}"><i class="fas fa-greater-than pr-2"></i>Organizing Projects</a></h3>
-        <p>Best practices for organizing your Pulumi programs.</p>
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts/state" >}}"><i class="fas fa-file-alt pr-2"></i>State and Backends</a></h3>
+        <p>Learn how Pulumi stores state and manages concurrency.</p>
     </div>
     <div class="w-1/2 border-solid ml-4 border-t-2 border-gray-200">
-        <h3 class="no-anchor pt-4"><a href="{{< relref "how-pulumi-works" >}}"><i class="fas fa-keyboard pr-2"></i>How Pulumi Works</a></h3>
-        <p>A peek under the hood to learn more about how Pulumi performs deployments.</p>
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts/how-pulumi-works" >}}"><i class="fas fa-upload pr-2"></i>How Pulumi Deploys Infrastructure</a></h3>
+        <p>Learn how Pulumi performs deployments.</p>
     </div>
 </div>
 <div class="md:flex flex-row mt-6 mb-6">
     <div class="w-1/2 border-solid border-t-2 border-gray-200">
-        <h3 class="no-anchor pt-4"><a href="{{< relref "state" >}}"><i class="fas fa-greater-than pr-2"></i>State and Backends</a></h3>
-        <p>An overview of how Pulumi stores state and manages concurrency.</p>
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts/config" >}}"><i class="fas fa-check-square pr-2"></i>Configuring Stacks</a></h3>
```

Comment:
Should "Configuring Stacks" now just be "Config"?  That fits better with most of the other section titles which are just the name of the concept like "Projects", "Resources", "Inputs and Outputs", etc.